README.org

```diff
@@ -201,5 +201,6 @@ webpaste first and then just read the documentation by running this:
  - [ ] paste.debian.net
  - [ ] bpaste.net
  - [ ] eval.in
+ - [X] bpa.st
```

Comment:
I do maintain this list to have the same ordering as the default order of the providers, also the numbers of list size and checked boxes is off with this new entry :slightly_smiling_face:
```suggestion
 - [X] bpa.st
 - [ ] paste.debian.net
 - [ ] bpaste.net
 - [ ] eval.in
```